feature-group-definitions/promise-withresolvers.yml

```diff
@@ -1,5 +1,5 @@
 name: Promise.withResolvers()
-description: Promise.withResolvers() static method
+description: The `Promise.withResolvers()` static method TODO.
```

Comment:
I am too tired to make sense of this one today, sorry.

Reply:
Live to fight another day!

Reply:
Here we are, another day!

```suggestion
description: The `Promise.withResolvers()` static method is an alternative to the `Promise()` constructor that returns the the promise and the resolution functions. You can use this to access `resolve and `reject` outside the scope of the executor function.
```

Reply:
Nice, I didn't know what this was, I might just use it some time.